contracts/RestrictedNORI.sol

```diff
@@ -504,18 +536,21 @@ contract RestrictedNORI is
   }
 
   /**
-   * @notice Claim released tokens and withdraw them to `recipient` address.
+   * @notice Claim sender's released tokens and withdraw them to `recipient` address.
    *
    * @dev This function burns `amount` of `RestrictedNORI` for the given schedule id
    * and transfers `amount` of `BridgedPolygonNORI` from the `RestrictedNORI` contract's
    * balance to `recipient`'s balance.
    *
-   * Enforcement of the availability of claimable tokens
-   * for the `_burn` call happens in `_beforeTokenTransfer`
+   * Enforcement of the availability of claimable tokens for the `_burn` call happens in `_beforeTokenTransfer`
    *
    * ##### Requirements:
    *
    * - Can only be used when the contract is not paused.
+   *
+   * @param recipient The address to which the underlying BridgedPolygonNORI will be sent
```

Comment:
This is super pedantic, but it's best practice to write in active voice for documentation. It helps with clarity and readability.

https://developers.google.com/style/voice